### ISSUE TITLE
[ui5-shellbar]: shellbar accessible name

### ISSUE THREAD

**Issue by JarrettChan** [SAP/ui5-webcomponents#8930]
### Bug Description

A new customer is complaining that the accessible name "Shell Bar" does not meet 508 accessibility requirements.

"We understand that the "Shell Bar" is the term for the SAP designed header. However, it doesn't meet 508 requirements and it doesn't provide any description for people using Accessibility apps. Is it possible that it will be changed to meet Accessibility requirements? "

Direct link to ticket in shared offline channel.

### Affected Component

ui5-shellbar

### Expected Behaviour

From discussing with our A11Y, this name is a subjective topic.
My guess is that we would need a hook to specify the aria label and provide the user a way to specify the label.
Please help determine the direction.

### Priority

None

### UI5 Web Components Version

1.17.4

### Browser

Chrome

### Organization

SuccessFactors

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by GerganaKremenska** [comment]
Hello @SAP/ui5-webcomponents-topic-p,

Please review the aria-label of Shell Bar and review if it necessary to add accessible-name.

Best Regards,
Gergana

**Comment by yanaminkova** [comment]
Hello @JarrettChan,

We are currently consulting with accessibility experts. 
Could you please specify which part of the standard is not being met by our current implementation? 
This information will help us provide a clearer explanation to the experts.

Thanks and Kind regards,
Yana

**Comment by JarrettChan** [comment]
The customer did not cite a specific standard. The customer disagrees that the accessible name "Shell Bar" is meaningful, which I guess is an inappropriate label/name.

**Comment by yanaminkova** [comment]
Hello @JarrettChan,

After consulting with the accessibility experts, we have confirmed that the ui5-shellbar aligns with both the design system and screen reader specifications. The default term "Shell bar" also adheres to the naming conventions outlined in the SAP Fiori Design Guidelines. According to feedback from the accessibility experts, altering this default term for users who are blind could result in a "semantic decoupling" in meaning.

For these reasons, we recommend not making any changes.

Thanks and Kind regards,
Yana
